### ISSUE TITLE
Unable to open build interface in cities

### ISSUE THREAD

**Issue by Eric-Tobias** [Azurency/CQUI_Community-Edition#353]
Using a city's buttons to either change the production, purchase with religion or coin, and assign workers to tiles, highlights the buttons but does not open the corresponding menu.

**To Reproduce**
1. Select a city you own.
2. On the bottom right hand corner, use either button to affect production or worked tiles.
4. See the buttons light up without a menu being shown.

**Expected behavior**
The corresponding menu opens.

**General informations**
 - OS: Windows 10 64 bit
 - Version [e.g. 1.0.0.262
 - Expansion: Rise and Fall
 - Other mods: Better Civilopedia, Better Report Screen (both disabled with bug still replicable).

**Comment by Azurency** [comment]
Thank you for your report. Can you post a save game along with the log files please ?  Without it I can't really help. You'll find them in `..\Documents\My Games\Sid Meier's Civilization VI\`

**Comment by Eric-Tobias** [comment]
I only had a quicksave which I have overwritten since. 

Investigations since I posted this point at the building queue as a likely culprit. Would you want me to try and get another save where it is replicable?

**Comment by PaulSlowey** [comment]
Hi Antoine, I've got the same issue. No districts were showing in the city build overlay and production selection was not available (either by clicking the current production on the city name or by the bottom right hand corner selection. I disabled all community mods, then put CQUI on first and a few others I didn't think would interfere. Problems continued. Let me know if there is anything else I can do to help get this working with other mods. :)
[POUNDMAKER 39 1720 BC.zip](https://github.com/Azurency/CQUI_Community-Edition/files/2450776/POUNDMAKER.39.1720.BC.zip)

**Comment by PaulSlowey** [comment]
Could it be running other mods that have similar functionality as CQUI? I was running **Map Tacs** (the one with unique buildings for each civ (_can I make that a suggestion for the next release?_) & **Production Que** (which I now note is part of CQUI. I switched those both off (and a few other mods) and it seems to be working fine now. I'll try adding Map Tacs and see if that stuffs everything again. :)

**Comment by Eric-Tobias** [comment]
> 
> 
> Could it be running other mods that have similar functionality as CQUI? I was running **Map Tacs** (the one with unique buildings for each civ (_can I make that a suggestion for the next release?_) & **Production Que** (which I now note is part of CQUI. I switched those both off (and a few other mods) and it seems to be working fine now. I'll try adding Map Tacs and see if that stuffs everything again. :)

I agree with @PaulSlowey that other mods seem to be the issue. I am not sure why or how, but it seems, for that game, I had some of the mods that are part of the CQUI installed separately. They didn't show up in the add-on menu but were still loaded. Had to manually remove them from some files and it all started to work again.

**Comment by idl0r** [comment]
Same problem here. No other mods. See my save (multiplayer, germany) here: http://qasl.de/~idl0r/tmp/peter.Civ6Save . There are several cities where I have that bug, not all though.

**Comment by idl0r** [comment]
```
Runtime Error: E:\Steam\steamapps\workshop\content\289070\1125169247\Assets\Expansion1\Replacements\ProductionPanel.lua:1898: operator / is not supported for number / nil
stack traceback:
	E:\Steam\steamapps\workshop\content\289070\1125169247\Assets\Expansion1\Replacements\ProductionPanel.lua:1898: in function 'PopulateList'
	E:\Steam\steamapps\workshop\content\289070\1125169247\Assets\Expansion1\Replacements\ProductionPanel.lua:564: in function 'View'
	E:\Steam\steamapps\workshop\content\289070\1125169247\Assets\Expansion1\Replacements\ProductionPanel.lua:2775: in function 'Refresh'
	E:\Steam\steamapps\workshop\content\289070\1125169247\Assets\Expansion1\Replacements\ProductionPanel.lua:445: in function 'OnCitySelectionChanged'
	[C]: in function 'func'
	[C]: in function '(anonymous)'
```

In this case, "qi.entry.ArmyCost" returns "nil"
